snap/snapcraft.yaml

```diff
@@ -1,12 +1,86 @@
 name: dcgm
 base: core24
 adopt-info: dcgm
-summary: Snap for NVIDIA DCGM and DCGM exporter
+summary: Snap for NVIDIA DCGM and DCGM-Exporter
 license: Apache-2.0
 contact: solutions-engineering@lists.canonical.com
 description: |
-  This snap includes NVIDIA DCGM and DCGM exporter to provide GPU monitoring via
-  Prometheus metrics. This snap does not include the dcgmproftester.
+  This snap includes NVIDIA DCGM and DCGM-Exporter to manage and monitor NVIDIA GPUs via the CLI or via Prometheus metrics.
+  For instance, th snap can be used to collect the metrics and make Grafana dashboard for data visualition.
```

Comment:
`th` -> `the`
`visualition` -> `visualization`

Reply:
If you use vscode I recommend installing the code spell checker. It helps me getting those small typos